charts/agent/charts/deployment-cluster-events/values.yaml

```diff
@@ -0,0 +1,149 @@
+deployment-cluster-events:
+  mode: deployment
+
+  nameOverride: "deployment-cluster-events"
+  namespaceOverride: "k8sexplorer"
+
+  image:
+    repository: observe-agent
+    # Overrides the image tag whose default is the chart appVersion.
+    # The service's name will be applied to the end of this value.
+    tag: "0.9.0"
+    pullPolicy: IfNotPresent
+
+  command:
+    name: "observe-agent"
+    extraArgs: ["start", "--config=/conf/observe-agent.yaml", "--otel-config=/conf/relay.yaml"]
+
+  serviceAccount:
+    create: false
+    name: "observe-agent-service-account"
+
+  extraVolumes:
+    - name: "observe-agent-deployment-config"
+      configMap:
+        name: "observe-agent"
+        items:
+          - key: "relay"
+            path: "observe-agent.yaml"
+        defaultMode: 420
+
+  extraVolumeMounts:
+    - name: observe-agent-deployment-config
+      mountPath: /conf
+
+  livenessProbe:
+    httpGet:
+      port: 13133
+      path: /status
+    initialDelaySeconds: 10
+    periodSeconds: 5
+  readinessProbe:
+    httpGet:
+      port: 13133
+      path: /status
+    initialDelaySeconds: 10
+    periodSeconds: 5
+
+  networkPolicy:
+    enabled: true
+    egressRules: [{}]
+
+  resources:
+    requests:
+      cpu: 250m
+      memory: 256Mi
+  podAnnotations: {
+    observe_monitor_purpose: observecollection,
+    observe_monitor_scrape: 'true',
+    observe_monitor_path: '/metrics',
+    observe_monitor_port: '8888',
+  }
+  ports:
+    metrics:
+      # The metrics port is disabled by default. However you need to enable the port
+      # in order to use the ServiceMonitor (serviceMonitor.enabled) or PodMonitor (podMonitor.enabled).
+      enabled: true
+      containerPort: 8888
+      servicePort: 8888
+      protocol: TCP
+  # this init container provides the cluster uid (kube-system namespace) as config map
+  initContainers:
+    - name: kube-cluster-info
```

Comment:
I doubt this is needed. `k8s.cluster.uid` should already be populated by the k8sattributes processor.